src/datasources/system/SystemDataSource.ts

```diff
@@ -53,20 +60,24 @@ export class SystemDataSource extends DataSourceApi<SystemQuery> {
           ],
         });
       } else {
-        let metadataResponse = await getBackendSrv().post<{ data: SystemMetadata[] }>(this.baseUrl + '/query-systems', { projection: this.transformProjection(defaultProjection) });
-        
+        const resolvedId = getTemplateSrv().replace(target.systemId);
```

Comment:
```suggestion
        const resolvedId = getTemplateSrv().replace(target.systemId, options.scopedVars);
```

I'm not exactly sure when this is necessary, but it's part of the data source examples, so we should follow it.